qutebrowser/browser/hints.py

```diff
@@ -602,13 +604,15 @@ def _start_cb(self, elems):
                        star_args_optional=True, maxsplit=2)
     @cmdutils.argument('win_id', win_id=True)
     def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
-              *args, win_id, mode=None):
+              *args, win_id, mode=None, chronicle=False):
```

Comment:
You're right (and I'm non-native as well), but I've ran into a conflict when trying "--history" earlier and I've thought that I should just think up some other name :slightly_smiling_face:  For the time being I've set up --history with -s flag (hi**s**tory). Probably not the best way - I've been thinking about putting just --history without a short version, but haven't found a way to do it.

Here are logs from the -h conflict:
```
15:19:01 VDEBUG   commands   command:_inspect_func:267 Adding arg history of type <class 'bool'> -> add_argument('--history', '-h', action='store_true', default=False, dest='history', help='Whether to add spawned or yanked link to the browsing history.\n')
15:19:01 WARNING  py.warnings imp:<module>:33 /usr/lib/python3/dist-packages/apport/report.py:13: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import fnmatch, glob, traceback, errno, sys, atexit, locale, imp

Traceback (most recent call last):
  File "./qutebrowser.py", line 29, in <module>
    sys.exit(qutebrowser.qutebrowser.main())
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/qutebrowser.py", line 161, in main
    from qutebrowser import app
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/app.py", line 45, in <module>
    from qutebrowser.completion.models import instances as completionmodels
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/completion/models/instances.py", line 30, in <module>
    from qutebrowser.completion.models import miscmodels, urlmodel, configmodel
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/completion/models/miscmodels.py", line 24, in <module>
    from qutebrowser.browser import browsertab
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/browser/browsertab.py", line 32, in <module>
    from qutebrowser.browser import mouse, hints
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/browser/hints.py", line 331, in <module>
    class HintManager(QObject):
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/browser/hints.py", line 608, in HintManager
    *args, win_id, mode=None, history=False):
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/commands/cmdutils.py", line 134, in __call__
    handler=func, **self._kwargs)
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/commands/command.py", line 155, in __init__
    self._inspect_func()
  File "/home/rahid/Programowanie/Repos/forks/qutebrowser/qutebrowser/commands/command.py", line 268, in _inspect_func
    self.parser.add_argument(*args, **kwargs)
  File "/usr/lib/python3.5/argparse.py", line 1353, in add_argument
    return self._add_action(action)
  File "/usr/lib/python3.5/argparse.py", line 1716, in _add_action
    self._optionals._add_action(action)
  File "/usr/lib/python3.5/argparse.py", line 1557, in _add_action
    action = super(_ArgumentGroup, self)._add_action(action)
  File "/usr/lib/python3.5/argparse.py", line 1367, in _add_action
    self._check_conflict(action)
  File "/usr/lib/python3.5/argparse.py", line 1506, in _check_conflict
    conflict_handler(action, confl_optionals)
  File "/usr/lib/python3.5/argparse.py", line 1515, in _handle_conflict_error
    raise ArgumentError(action, message % conflict_string)
argparse.ArgumentError: argument --history/-h: conflicting option string: -h
```